site/components/pages/Infinity/Cards/Card.tsx

```diff
@@ -16,38 +16,41 @@ type Card = {
   description: string;
   tonsRetired: number;
   date: string;
+  width?: number;
+  height?: number;
 };
 
 type Props = {
   card: Card;
 };
 
-export const Card: FC<Props> = ({ card }) => {
+export const Card: FC<Props> = (props) => {
```

Comment:
Why do you prefer to not destructure props?
Saw that in another PR of yours too.
Just curious.

Reply:
NVM, let's see what Atmos says [here](https://github.com/KlimaDAO/klimadao/pull/591#discussion_r947505002)

Reply:
@ladytrekker it's the law 👮‍♂️ 

https://github.com/KlimaDAO/klimadao/wiki#do-not-destructure-props